docs/source/tutorials/quick-start.rst

```diff
@@ -110,6 +110,43 @@ The inference command is:
 
 This inference command predicts the classes of nodes in the testing set and saves the results, a Pytorch tensor file named "**predict-0.pt**", into the ``/tmp/ogbn-arxiv-nc/predictions/`` folder.
 
+Generating Embedding
+------------------------
+If users only need to generate node embeddings instead of doing predictions on the graph, users can use saved model and the same yaml configure file used in training to achieve that with the ``gs_gen_node_embedding`` command:
+.. code-block:: bash
+
+    python3 -m graphstorm.run.gs_gen_node_embedding \
+               --workspace /tmp/ogbn-arxiv-nc \
+               --num-trainers 1 \
+               --part-config /tmp/ogbn_arxiv_nc_1p/ogbn-arxiv.json \
+               --ip-config /tmp/ogbn-arxiv-nc/ip_list.txt \
+               --ssh-port 2222 \
+               --save-embed-path /tmp/saved_embed \
+               --restore-model-path /tmp/ogbn-arxiv-nc/models/epoch-7/ \
+               --cf /graphstorm/training_scripts/gsgnn_np/arxiv_nc.yaml \
+               --use-mini-batch-infer true
+
+Users need to specify ``--restore-model-path`` and ``--save-embed-path`` when using the command above to generate node embeddings, and the node embeddings will be saved into the save-embed path.
+
+For node classification/regression task, if ``target_ntype`` is provided, the command will generate and save node embeddings on ``target_ntype``, otherwise it will generate embeddings of all node types.
```

Comment:
Sorry, my fault. 
Could you change "embeddings of all node types" to "embeddings for all node types"?